### PR TITLE
Modify the Stripe JS to better adhere to our Standards

### PR DESCRIPTION
**Pull request by bordoni**
### 🗒️ Description

Took a stab at the Code from to make the JS follow the ES6 standards that we are trying to move to and some small fixes that were triggering Notices.

### CODE REVIEW

File: src/resources/js/commerce/gateway/stripe/checkout.js [hunk 1]
```diff
@@ -206,60 +202,84 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
...
+	 * @since TBD
+	 *
+	 * @param result
+	 */
+	obj.handleConfirmPayment = ( result ) => {
+		obj.submitButton( true );
+		if ( result.error ) {
+			obj.handlePaymentError( result );
+		} else {
+			// @todo feels like we are missing a error handling here.
```
Comment (moraleida) [inline src/resources/js/commerce/gateway/stripe/checkout.js:240]: no exactly error handling, but some cases in which the payment method returns a partial success, like a status of `pending_confirm` and some others.

**Review by moraleida — APPROVED** [review]
Great stuff. Some of these changes are also in #2264 so it would be good to merge this before that one, I think

File: src/resources/js/commerce/gateway/stripe/checkout.js [hunk 2]
```diff
@@ -377,4 +424,4 @@ tribe.tickets.commerce.gateway.stripe.checkout = {};
 	};
 
 	$( obj.ready );
-})( jQuery, tribe.tickets.commerce, Stripe, ky );
+})( jQuery, tribe.tickets.commerce, Stripe, tribe.ky );
```
Comment (juanfra) [inline src/resources/js/commerce/gateway/stripe/checkout.js:427]: The object here should be `tribe.tickets.commerce.gateway.stripe.checkout`

**Review by juanfra — COMMENTED** [review]
Changes look good!

File: src/resources/js/commerce/gateway/stripe/checkout.js (hunk 2 above)
Comment (moraleida) [reply src/resources/js/commerce/gateway/stripe/checkout.js:427]: @juanfra this fix was edded to #2264

**Pull request merged**